.flake8

```diff
@@ -188,6 +188,7 @@ extend-ignore =
   WPS613  # FIXME: unmatching super method access
   WPS615  # FIXME: unpythonic setter/getter
   PT027   # FIXME: use pytest.raises() instead of unittest-style 'assertRaises'
+  S507    # FIXME: Paramiko call with policy set to automatically trust the unknown host key
```

Comment:
It's best to ignore such exceptions inline instead of allowing the violations globally. This way you're in control and have to review the security implications of every such place.